bndtools.core.test/bnd.bnd

```diff
@@ -0,0 +1,52 @@
+# Set javac settings from JDT prefs
+-include: ${workspace}/cnf/includes/jdt.bnd, ${workspace}/cnf/includes/bndtools.bnd
+-sub: *.bnd
```

Comment:
Since this is a test project and should not be released, you need to make sure it is not released.

```properties
-buildrepo:
-releaserepo:
```